README.md

```diff
@@ -64,7 +64,7 @@ class UserSerializer < BaseSerializer
   # The previous one is a shorthand for the following lines:
   lazy_relationship :blog_posts, loader: AmsLazyRelationships::Loaders::Association.new("User", :blog_posts)
   has_many :blog_posts, serializer: BlogPostSerializer do |serializer|
-    serializer.lazy_blog_posts
+    -> { serializer.lazy_blog_posts }
```

Comment:
It can still be called without using a proc, right?
Maybe we could keep the old version for simplicity and just mention the proc form in the `include_data` example you added in https://github.com/Bajena/ams_lazy_relationships/pull/47/files#diff-04c6e90faac2675aa89e2176d2eec7d8R134 ?

```suggestion
    serializer.lazy_blog_posts
```

Reply:
> It can still be called without using a proc, right?

sure, Reflection monkey-patch will forward non-proc value as it is.
But I believe that in readme is better to propose the best possible sample by default.
Just imagine:
1) engineer_1 adds non-proc custom finders all over the services & serializers;
2) engineer_2 turns `ActiveModel::Serializer.config.include_data_default = false` globally
3) local naive tests will assert successfully that json response is free from `data`, but
4) production servers will continue to fail due to OOM and that will be quite tricky to discover why

I think we need to leave `-> { serializer.lazy_blog_posts }` as a very default sample, and add a small hint that `serializer.lazy_blog_posts` will work as well but can produce redundant queries.

Reply:
> I think we need to leave -> { serializer.lazy_blog_posts } as a very default sample, and add a small hint that serializer.lazy_blog_posts will work as well but can produce redundant queries.

Ok, I must admit that your explanation makes sense to me. Let's leave proc, but also let's mention the non-proc version as you suggested 👍

Reply:
roger that! )